src/formatDate.js

```diff
@@ -8,7 +8,29 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arr = date.split(fromFormat[fromFormat.length - 1]);
+  const obj = {};
```

Comment:
The variable name `obj` is not descriptive. Consider renaming this variable to something more meaningful, such as `dateComponents`.